### ISSUE TITLE
Unable to change divider color

### ISSUE THREAD

**Issue by Aravin** [saadeghi/daisyui#254]
Unable to change divider color

**Comment by Aravin** [comment]
Coool. Thanks @saadeghi

**Comment by MentalGear** [comment]
Also struggled with this for a while. Could you please add the examples to the docs @saadeghi ? (would have done it, but docs seem not to be one the repo)

> You can change text color using text-* color utilities.
> You can also change line colors using before:bg-* and after:bg-* color utilities (JIT needed)

**Comment by MentalGear** [comment]
Came back here because I again struggled with this as my divider was the same color as the background ie  invisible; changing color isn't yet described in the divider doc page.
The method described in the playground by @saadeghi seems to not work anymore.

**Comment by saadeghi** [comment]
@MentalGear Here's a new example: https://play.tailwindcss.com/B7X71XGhTx  
I will add it to the docs soon

**Comment by MentalGear** [comment]
Thank you for the quick responsiveness!

**Comment by myleftshoe** [comment]
How to change divider color when in dark mode? I'm using DaisyUI with `theme-change` so prefixing with `dark:` doesn't work. 
I could I guess read the theme name from localStorage and conditionally render but that's seems overkill just to change divider color.

**Comment by myleftshoe** [comment]
Using divide-base-300 works well enough!

**Comment by saadeghi** [comment]
@myleftshoe Yes, If you use one of [daisyUI color names](https://daisyui.com/docs/colors/), the automatically adapt the dark/light theme. no prefix or config needed.

**Comment by myleftshoe** [comment]
Auto adapting color scheme is a great feature. Tailwind claims dark mode support out-of-the-box but it isn't really, vanilla css could claim the same. DaisyUI is truly out-of-the-box and one of the main reasons I chose it - who wants to spend time messing around prefixing class names with "dark" and continually tweaking when you miss something. Thanks!

**Comment by mcfry** [comment]
Still needs added
